connectome_interpreter/external_map.py

```diff
@@ -65,9 +69,13 @@ def load_dataset(dataset):
         data = pkgutil.get_data(
             "connectome_interpreter", "data/Dweck2018/larva_fruit2or.csv"
         )
+    elif dataset == "Nern2024":
+        data = pkgutil.get_data(
+            "connectome_interpreter", "data/Nern2024/ME-columnar-cells-hex-location.csv"
+        )
     else:
         raise ValueError(
-            "Dataset not recognized. Please choose from 'DoOR_adult', 'DoOR_adult_sfr_subtracted', 'Dweck_adult_chem', 'Dweck_adult_fruit', 'Dweck_larva_chem', 'Dweck_larva_fruit'."
+            "Dataset not recognized. Please choose from 'DoOR_adult', 'DoOR_adult_sfr_subtracted', 'Dweck_adult_chem', 'Dweck_adult_fruit', 'Dweck_larva_chem', 'Dweck_larva_fruit', 'Nern2024'."
```

Comment:
This is fine, but I'll do a PR to put these values in a dictionary to make it a bit easier to maintain.

Reply:
See #12